lib/Stats.js

```diff
@@ -405,16 +417,16 @@ Stats.jsonToString = function jsonToString(obj, useColors) {
 			colSizes[col] = 3;
 		for(row = 0; row < rows; row++) {
 			for(col = 0; col < cols; col++) {
-				value = array[row][col] + "";
+				value = getValue(array, row, col) + "";
 				if(value.length > colSizes[col]) {
 					colSizes[col] = value.length;
 				}
 			}
 		}
 		for(row = 0; row < rows; row++) {
 			for(col = 0; col < cols; col++) {
-				var format = row === 0 ? colors.bold : formats[col];
-				value = array[row][col] + "";
+				var format = row === 0 ? colors.bold : array[row][col].color;
```

Comment:
@TheLarkInn @sokra I couldn't able to make the stats work the way we wanted with the existing data structure, so added this tiny hack.. But the other solution would be to check if the value is truthy and change the colour according to the input.. 

I have this solution done here - https://github.com/vigneshshanmugam/perf-hints-plugin/blob/master/src/CustomStats.js#L95